tests/unit/hosts/test_models.py

```diff
@@ -0,0 +1,50 @@
+from django.test import TestCase
+from django.contrib.auth.models import User
+
+from hosts.models import ChallengeHost, ChallengeHostTeam
+
+
+class BaseTestCase(TestCase):
+
+    def setUp(self):
+        self.user = User.objects.create(
+            username='dfsdf',
```

Comment:
Making the test cases consistent with the other  test  cases, can you please make sure that you change the username to `user` instead of `dfsdf`